app/Components/Icon/Icon.tsx

```diff
@@ -0,0 +1,19 @@
+import Image from 'next/image';
+import { IconNameEnum } from './enums/icon-name.enum';
+
+interface Props {
+  name: IconNameEnum;
+  isActive?: boolean;
+  width: number;
+  height: number;
+}
+
+const Icon: (props: Props) => JSX.Element = (props: Props) => {
```

Comment:
Can we take this (props: Props) => JSX.Element and interface props in its own file?